integration-tests/pages/calendarDetailPage.js

```diff
@@ -0,0 +1,12 @@
+const page = require('./page')
+
+const calendarDetailPage = (date) =>
+  page(date, {
+    detailStart: () => cy.get('.detail-start'),
+    previousDay: () => cy.get('#btnPreviousShiftDetail'),
+    nextDay: () => cy.get('#btnNextShiftDetail'),
+  })
+
+module.exports = {
```

Comment:
I had to use `module.exports` instead of `export default`, I'm sure someone who knows about node will be able to fix it for me :grin:

Reply:
Does the test code run using the node engine or a browser specific one?

Reply:
Don't think we did anything special for this..